pkg/setting/settings_zanzana.go

```diff
@@ -48,6 +46,54 @@ type ZanzanaServerSettings struct {
 	SigningKeysURL string
 }
 
+// Parameters to configure OpenFGA cache.
+type OpenFgaCacheSettings struct {
```

Comment:
I wonder if some of these are better left unconfigurable for now.  Once we allow configuration of these, we need to support configuration of these forever lest we break someone's workflow.

We can always add configuration points later and I agree that _some_ control over the caching makes sense, but this is a lot of granularity that we likely can just decide what the best value is until someone says different.

Reply:
I agree that it does feel a bit premature to expose all these options, as we're not using most of them yet.

I'm not too worried about customers using them and us needing to support the options because of that - as long as we don't document or advertise these config options, we can treat them as internal only. So if we feel like we'll need these options, I'm fine to merge them. Just feels a bit overwhelming for now 😄